src/tests/dictionaryConcepts/container/CreateConcept.test.jsx

```diff
@@ -253,4 +253,43 @@ describe('Test suite for dictionary concepts components', () => {
       expect(wrapper.find('CreateConceptForm').props().mappings[0].map_type).toEqual(inputValue);
     });
   });
+
+  it('should follow the provided path when the Leave button is clicked', () => {
+    const mockPath = localStorage.getItem('dictionaryPathName');
+    const instance = wrapper.find('CreateConcept').instance();
```

Comment:
Why are you repeating 'CreateConcept' in several places?

Reply:
I have created a variable to avoid this repetition